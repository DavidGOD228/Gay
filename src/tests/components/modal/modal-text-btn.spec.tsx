import 'react-native'
// Note: test renderer must be required after react-native.
import React from 'react'
import renderer from 'react-test-renderer'
import ModalTextBtn from '../../../components/modal/modal-text-btn'
import {
  Text
} from 'react-native'

it('renders correctly empty', () => {
  const tree = renderer.create(
    <ModalTextBtn onPress={() => null}/>
  );
  expect(tree).toMatchSnapshot();
})

it('renders correctly text', () => {
  const tree = renderer.create(
    <ModalTextBtn onPress={() => null}>
      <Text>
        All you had to do was follow the damn train CJ
      </Text>
    </ModalTextBtn>
  );
  expect(tree).toMatchSnapshot();
})