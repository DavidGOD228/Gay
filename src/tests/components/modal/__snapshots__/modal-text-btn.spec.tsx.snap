// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly empty 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#6B8090",
          "fontSize": 18,
          "fontWeight": "bold",
        }
      }
    />
  </View>
</View>
`;

exports[`renders correctly text 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#6B8090",
          "fontSize": 18,
          "fontWeight": "bold",
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
      >
        All you had to do was follow the damn train CJ
      </Text>
    </Text>
  </View>
</View>
`;
